Server/User/database.py

```
@@ -31,4 +40,11 @@ def get_db():
     try:
         yield db
     finally:
-        db.close() 
\ No newline at end of file
+        try:
+            db.close()
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)